session.go

```diff
@@ -1,97 +1,99 @@
 package bond
 
 import (
+	"database/sql"
+	"fmt"
 	"reflect"
 	"sync"
 
-	"upper.io/db"
+	"upper.io/db.v2"
+	"upper.io/db.v2/lib/sqlbuilder"
```

Comment:
Makes sense


On July 28, 2016 at 3:47:05 PM, José Carlos (notifications@github.com)
wrote:

In session.go <https://github.com/upper/bond/pull/20#discussion_r72689488>:

>    "reflect"
>    "sync"
>
> -  "upper.io/db"
> +  "upper.io/db.v2"
> +  "upper.io/db.v2/lib/sqlbuilder"

We're importing sqlbuilder here because it provides interfaces for creating
SQL adapters, in fact these interfaces for SQL adapters can live anywhere,
even on db.go the only problem is that they require importing database/sql,
that's why I moved them away from db.go.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/upper/bond/pull/20/files/71a7791afe21ffdef008ca2073c9efc45e3d463c#r72689488>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AABJjzWRJ6GIRNzdB0HCZYv-AQQsCzuEks5qaQc5gaJpZM4JEcwJ>
.